elixir/apps/web/lib/web/live/policies/edit.ex

```diff
@@ -5,7 +5,11 @@ defmodule Web.Policies.Edit do
 
   def mount(%{"id" => id}, _session, socket) do
     with {:ok, policy} <- Policies.fetch_policy_by_id(id, socket.assigns.subject) do
-      {:ok, assign(socket, policy: policy)}
+      {:ok,
+       assign(socket,
+         policy: policy,
+         form: to_form(Policies.Policy.Changeset.update(policy, %{}))
+       )}
```

Comment:
The form can be temporarily assigned to reduce the state that websocket will hold, since we will rebuild it on every update anyways:
```suggestion
      form = to_form(Policies.Policy.Changeset.update(policy, %{}))
      {:ok, assign(socket, policy: policy), temorary_assigns: [form: form]}
```

Reply:
👍 makes sense.  I've update this and the Policy `new.ex` liveview with the temporary assigns.

Also, I tried the suggested code as is, but it did not work.  It looks like you need to assign it to the socket using the `assign` function and then in the `temporary_assigns` value you tell it which assigns are temporary and what the value of the assign should be reset to.